Workbooks/WorkbooksMetadata.json

```diff
@@ -306,7 +306,7 @@
     "dataConnectorsDependencies": [],
     "previewImagesFileNames": [ "AzureNetworkWatcherWhite.png", "AzureNetworkWatcherBlack.png" ],
     "version": "1.1",
-    "title": "Azure Network Watcher",
+    "title": "Network Watcher Traffic Analtics",
```

Comment:
typo - analytics?

Reply:
Hi @liatlishams - please look into this and if still applicable fix / approve / merge and if not applicable please close the PR. This is an old one. Thanks.